packages/design-system/src/components/N8nCallout/Callout.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+	<div :class="classes" role="alert">
+
+		<div :class="$style['message-section']">
+			<div :class="$style.icon">
+				<n8n-icon
+					v-bind="$attrs"
```

Comment:
We should use `v-bind="$attrs"` only on a root-level element, because this will assign all external attributes to the element and can lead to duplicate ids, repeating class names and unwanted attributes.